IMAGE:
  WIDTH: 640
  HEIGHT: 480
  ROI_START_HEIGHT: 360
  ROI_HEIGHT: 30

XYCAR:
  START_SPEED: 5.0
  MAX_SPEED: 10.0
  MIN_SPEED: 5.0
  SPEED_CONTROL_THRESHOLD: 30.0
  ACCELERATION_STEP: 0.05
  DECELERATION_STEP: 0.1

PID:
  P_GAIN: 0.40
  I_GAIN: 0.00000
  D_GAIN: 0.00
  
MOVING_AVERAGE_FILTER:
  SAMPLE_SIZE: 20

TOPIC:
  PUB_NAME: /xycar_motor
  SUB_NAME: /usb_cam/image_raw/
  QUEUE_SIZE: 1

CANNY:
  LOW_THRESHOLD: 100
  HIGH_THRESHOLD: 200

HOUGH:
  ABS_SLOPE_RANGE: 40.0
  THRESHOLD: 20
  MIN_LINE_LENGTH: 10
  MAX_LINE_GAP: 5

DEBUG: false
